element.go

```diff
@@ -186,3 +193,19 @@ func splitAndVerifyElementSequence(s string) (arguments string, elementType int,
 
 	return
 }
+
+func parseBuildkiteElementSequence(sequence string) (*element, error) {
+	// Expect bk;t=123123234234234
+	// TODO: Properly handle parsing multiple arguments
+
+	if !strings.HasPrefix(sequence, "bk;t=") {
```

Comment:
If we match for `"bk;"` - could we pass through all the k/v pairs and render each of them into one of the `<?` nodes?